src/main/java/org/apache/commons/collections4/trie/AbstractPatriciaTrie.java

```diff
@@ -341,10 +341,6 @@ public V selectValue(final K key) {
         return entry.getValue();
     }
 
-    /**
-     * This is equivalent to the other {@link #selectR(TrieEntry, int, Object, int, Cursor, Reference)}
-     * method but without its overhead because we're selecting only one best matching Entry from the {@link Trie}.
-     */
```

Comment:
I think this one needs to stay?

Reply:
> I think this one needs to stay?

I delete it because I can never found that function it point to.

Reply:
Ah, you are correct! There was another method, but it was removed when the base class was renamed and the code changed a little bit to create this abstract class - https://github.com/apache/commons-collections/commit/83226e162de60bd22551b1a37cc3e513811e1449#diff-821b6e7d45ec0fd696f3cdc8027f029cL303

All good here then. Thanks!